regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -82,7 +82,7 @@ var ChildView = Backbone.View.extend({
     // for the fixed position header, is deemed the active section
     checkActiveSection: function() {
       $.each(this.$sections, function(idx, $section) {
-        if ($section.offset().top + this.$contentHeader.height() >= $(window).scrollTop()) {
+        if ($section.offset().top >= $(window).scrollTop()) {
```

Comment:
It looks like this is still needed:

Previous:
<img width="658" alt="screen shot 2016-06-20 at 10 40 38 am" src="https://cloud.githubusercontent.com/assets/326918/16198327/89901688-36d3-11e6-8867-9e9fd34d9657.png">


With Patch:
<img width="659" alt="screen shot 2016-06-20 at 10 40 52 am" src="https://cloud.githubusercontent.com/assets/326918/16198324/83df1676-36d3-11e6-9769-d82a3b36d94a.png">